### PR TITLE
Update to the config Dev UI screen

### LINKED ISSUE #14933: Dev UI config editor does not make it clear which property is set and which is not
There is the blue background but it's not made explicit what it is about

### PR DESCRIPTION
**Pull request by phillip-kruger**
This PR 

fix #14933
by:
- Group properties by Config source (so its easy to see what has been added to `application.properties`)
- Disable editing of Env and sys config values

109828476-54123880-7c45-11eb-8777-f2222170e606

Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]
LGTM

File: extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigSourceName.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
+package io.quarkus.vertx.http.runtime.devmode;
+
+import java.util.Objects;
+
+public class ConfigSourceName implements Comparable<ConfigSourceName> {
+    private static final String APPLICATION_PROPERTIES_CONFIG_SOURCE = "PropertiesConfigSource[source=application.properties]";
+    private static final String SYSTEM_PROPERTIES_CONFIG_SOURCE = "SysPropConfigSource";
+    private static final String ENVIRONMENT_CONFIG_SOURCE = "EnvConfigSource";
+    private static final String PROPERTIES_CONFIG_SOURCE = "PropertiesConfigSource";
+    private static final String DEFAULT_VALUES = "default values";
```
Comment (gsmet) [inline extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigSourceName.java:10]:
```suggestion
    private static final String DEFAULT_VALUES = "Default values";
```
would be better given it's the title of a section.

**Review by gsmet — DISMISSED** [review]
I added a minor comment. Otherwise it looks good!

File: extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigSourceName.java (hunk 1 above)
Comment (phillip-kruger) [reply extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigSourceName.java:10]: It's called `default values` in config, but I can change the display name to be `Default values`. That is what you want right ?

Comment (phillip-kruger) [reply extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigSourceName.java:10]: (So the config source name is `default values`)

Comment (phillip-kruger) [reply extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigSourceName.java:10]: Done. Will now show on the screen as `Default value`.

**Comment by phillip-kruger** [comment]
@gsmet ci failure seems unrelated ?

**Review by radcortez — APPROVED** [review]

**Comment by geoand** [comment]
The CI failure is totally unrelated.

**Pull request merged**